Assets/Scripts/Enemy/EnemySpawn.cs

```diff
@@ -194,11 +194,16 @@ void SpawnEnemy(EnemyType enemy)
             enemyType = enemy;
         }
 
-        random = Random.Range(0, 10);
-        if (enemyType == EnemyType.TURRET && random < 5)
+        // 50% to be predictive if turret or ranged.
+        if ((enemyType == EnemyType.TURRET || enemyType == EnemyType.RANGED) && Random.Range(0, 2) == 0)
+        {
+            variant = EnemyVariantType.PREDICTIVE;
+        }
+        else if (enemyType == EnemyType.TURRET && Random.Range(0, 2) == 0)
```

Comment:
max is gonna scream

Reply:
Got a shiny new ticket for this haha:
https://github.com/MaXeraph/Adrenarun/issues/93

Reply:
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa